src/Resources/Products/Suggest.ts

```diff
@@ -0,0 +1,13 @@
+import AbstractResource from '../AbstractResource';
+
+export default class Suggest extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '';
+        this.defaultAccess = 'guest';
+    }
+
+    public all: Function = this.createMethodFromPartialSpec({
+        method: 'GET',
+        path: '/products/suggest',
```

Comment:
I think better to have this in the `this.resourcePath` above for in case more methods are added later on